torchgeo/models/rcf.py

```diff
@@ -73,9 +95,85 @@ def __init__(
             ),
         )
         self.register_buffer(
-            "biases", torch.zeros(features // 2, requires_grad=False) + bias
+            "biases", torch.zeros(num_patches, requires_grad=False) + bias
+        )
+
+        if mode == "empirical":
+            assert dataset is not None
```

Comment:
This line can be dropped, we already check this above

Reply:
mypy complains

Reply:
Ah, because it's a string value thing, not catchable by a static type analyzer.